authentication/views/auth.py

```diff
@@ -60,10 +69,15 @@ def login_view(request):
 
 def activation_view(request, code: str):
     if request.method == "POST":
-        form = ActivationForm(user=None, data=request.POST)
+        form = CustomSetPasswordForm(user=None, data=request.POST)
         activation_record = get_object_or_404(Activation, code=code)
         form.user = activation_record.user
-        code_is_valid(code, request.POST["email"], activation_record.user)
+        code_is_valid(
+            code,
+            request.POST["email"],
+            activation_record.user,
+            settings.ACTIVATION_EXPIRATION_DAYS,
+        )
```

Comment:
It doesn't appear that this function result is being checked.

Reply:
Good catch here, I've added code to fix this gap.